integreat_cms/static/src/js/push-notifications.ts

```diff
@@ -10,10 +10,20 @@ const switchLanguage = (language: string) => {
     document.getElementById(`li-${language}`).classList.add("z-10", "text-blue-500", "bg-white", "cursor-default");
 };
 
+const adjustTextDirection = (textDirection: string, language: string) => {
+    if (textDirection === "RIGHT_TO_LEFT") {
+        const input = document.getElementById(`tab-${language}`).getElementsByTagName("input")[0] as HTMLElement;
+        const textarea = document.getElementById(`tab-${language}`).getElementsByTagName("textarea")[0] as HTMLElement;
+        input.dir = "rtl";
+        textarea.dir = "rtl";
+    }
+};
+
 window.addEventListener("load", () => {
     document.querySelectorAll("[data-switch-language]").forEach((node) => {
         (node as HTMLElement).addEventListener("click", () => {
             switchLanguage(node.getAttribute("data-switch-language"));
+            adjustTextDirection(node.getAttribute("text-direction"), node.getAttribute("data-switch-language"));
         });
```

Comment:
Since I have to convert node to an HTMLElement for this to work, it's longer. Do still like it better via dataset? I don't really have a preference
![grafik](https://user-images.githubusercontent.com/72705147/228522407-1eb65107-6b0a-4939-8036-f91e5b61302a.png)

Reply:
Change has been applied